libmscore/instrument.cpp

```diff
@@ -213,6 +214,7 @@ void Instrument::write(XmlWriter& xml, const Part* part) const
       _shortNames.write(xml, "shortName");
 //      if (!_trackName.empty())
             xml.tag("trackName", _trackName);
+      xml.tag("nameColor", _nameColor);
```

Comment:
This seems the cause for the Travis CI mtest failures:
Only write when not the default (black I guess), can be done by passing the default as a 3rd paramenter to that method IIRC

Reply:
instead of black maybe whatever color is set to be the style for instrument text?